pkg/utils/utils.go

```diff
@@ -354,14 +362,16 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 			return nil, errors.New("Failed to retrieve S3 Secret Access Key.")
 		}
 		logger.Infof("Using explicitly found credentials for S3 repository access.")
-		sess = session.Must(session.NewSession(&aws.Config{
-			Region:      aws.String(region),
-			Credentials: credentials.NewStaticCredentials(s3AccessKeyId, s3SecretAccessKey, ""),
-		}))
-	} else {
-		sess = session.Must(session.NewSession(&aws.Config{
-			Region: aws.String(region),
-		}))
+		credential = credentials.NewStaticCredentials(s3AccessKeyId, s3SecretAccessKey, "")
+		sessionOptions.Config.Credentials = credential
+	}
+	caCert, ok := GetStringFromParamsMap(params, "caCert", logger)
+	if ok && len(caCert) > 0 {
+		sessionOptions.CustomCABundle = strings.NewReader(caCert)
+	}
+	sess, err := session.NewSessionWithOptions(sessionOptions)
```

Comment:
Let's keep using `session.Must`. If S3 session can be created successfully, the backup data cannot go anyway. It is necessary to draw users' attention using the panic.